back/app/services/permissions/user_requirements_service.rb

```diff
@@ -98,27 +98,31 @@ def mark_satisfied_requirements!(requirements, permission, user)
       when :confirmation
         !user.confirmation_required?
       when :group_membership
-        user.in_any_groups?(permission.groups)
+        permission.groups.present? && user.in_any_groups?(permission.groups)
       end
       requirements[:special][special_key] = 'satisfied' if is_satisfied
     end
   end
 
   def onboarding_possible?
-    app_configuration.settings.dig('core', 'onboarding') && (
-      Topic.where(include_in_onboarding: true).count > 0 || Area.where(include_in_onboarding: true).count > 0
-    )
+    return @onboarding_possible unless @onboarding_possible.nil?
+
+    @onboarding_possible = app_configuration.settings.dig('core', 'onboarding') && (Topic.where(include_in_onboarding: true).count > 0 || Area.where(include_in_onboarding: true).count > 0)
```

Comment:
Could not use `||=` here because it doesn't work when the value returned is false - so will always keep rerunning the queries